ui/imports/shared/status/StatusEmojiSuggestionPopup.qml

```diff
@@ -19,7 +19,7 @@ StatusInputListPopup {
     }
 
     getImageSource: function (modelData) {
-        return `qrc:/StatusQ/src/assets/twemoji/72x72/${modelData.unicode}.png`
+        return `qrc:/StatusQ/src/assets/twemoji/svg/${modelData.unicode}.svg`
```

Comment:
Would be good to keep Emoji handling in one place in the codebase.

```suggestion
        return Emoji.iconSource(modelData.unicode)
```

or you could introduce a new more performant method there to obtain image sources from emoji char, because in this case text parsing is not needed.

Reply:
Done, I introduced a new function, `Emoji.svgImage(unicode)`